src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -43,6 +43,16 @@ import { isStandalone } from 'vs/base/browser/browser';
 				'markdownDescription': nls.localize('highlightModifiedTabs', "Controls whether a top border is drawn on modified (dirty) editor tabs or not. This value is ignored when `#workbench.editor.showTabs#` is disabled."),
 				'default': false
 			},
+			'workbench.editor.tabDecorations.badges': {
```

Comment:
I feel the setting name `workbench.editor.tabDecorations` should be more generic because it also applies when tabs are disabled. Given we seem to call it `explorer.decorations` or `problems.decorations` elsewhere, why not `workbench.editor.decorations`? For example, we already have `workbench.editor.showIcons` which is targetting the editor label itself so I think we do not need to be more specific.

Reply:
I like that